.kokoro/requirements.txt

```diff
@@ -4,9 +4,9 @@
 #
 #    pip-compile --generate-hashes requirements.in
 #
-attrs==22.2.0 \
-    --hash=sha256:29e95c7f6778868dbd49170f98f8818f78f3dc5e0e37c0b1f474e3561b240836 \
-    --hash=sha256:c9227bfc2f01993c03f68db37d1d15c9690188323c067c641f1a35ca58185f99
+attrs==22.1.0 \
```

Comment:
@ddixit14 @suztomo Do you know why OwlBot is doing this downgrading? Is it copying templates? Why do we even need OwlBot on this repository?

Reply:
I don't have an idea about OwlBot functioning :/